modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/SettingsImpl.kt

```diff
@@ -544,6 +544,12 @@ class SettingsImpl @Inject constructor(
         sharedPrefs = sharedPreferences,
     )
 
+    override val useEpisodeArtwork = UserSetting.BoolPref(
+        sharedPrefKey = "useEpisodeArtwork",
+        defaultValue = false,
```

Comment:
The default value should be `true`. This is currently enabled for all of the users.